### PR TITLE
Update lib-descs.js

### PR DESCRIPTION
**Pull request by martinsjoborg**

### CODE REVIEW

**Review by olleolleolle — APPROVED** [review]
Thanks for keeping the graph updated!

File: lib-descs.js [hunk 1]
```diff
@@ -7,38 +7,46 @@
...
+  "            ": "pontus_gågatan" //a6d7b1efb918
+  "            ": "salong_gorgeous" //0e6e136470f8
+  "c025e9a84b32": "restsara_bergs" //9e0b3f4b48f0
+  "c025e99a0e3a": "hojen_friisg" //b618c21ad800
+  "c025e999e70e": "hojen_mölleg" //76069e4ed318
+  "8416f92da9e6": "carlings_sfg" //22f3a4948b80
+  "c025e9a84c8a": "cykelson_bergsg" //8a09b7e13818
+  "8416f92daa8e": "beauty.r.salong_fisk" //dab0b01603a8
+  "c025e999ea9a": "thaithai" //1ae441efb720
+  "c025e999e6ea": "thaithai" //46e7340fab78
```
Comment (olleolleolle) [inline lib-descs.js:49]: Men, här saknas komma.

**Review by olleolleolle — CHANGES_REQUESTED** [review]

File: lib-descs.js (hunk 1 above)
Comment (martinsjoborg) [reply lib-descs.js:49]: aj =)

Comment (quite) [reply lib-descs.js:49]: Merged this by hand.

**Pull request closed without merging**